src/base/http/server.h

```diff
@@ -31,13 +31,9 @@
 #ifndef HTTP_SERVER_H
 #define HTTP_SERVER_H
```

Comment:
@Chocobo1:  This should probably be converted to `#pragma once`.

Reply:
Due to http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#sf8-use-include-guards-for-all-h-files (the "note" part), I tend not to touch them when not necessary, of course I'll still use `#pragma once` for new files.